models/public/resnet-50-tf/resnet-50-tf.md

```diff
@@ -0,0 +1,85 @@
+# resnet-50-tf
+
+## Use Case and High-Level Description
+
+`resnet-50-tf` is a TensorFlow\* implementation of ResNet-50 - an image classification model
+pretrained on the ImageNet dataset. Originally redistributed in Saved model format,
+converted to frozen graph using `tf.graph_util` module.
+For details see [paper](https://arxiv.org/pdf/1512.03385.pdf),
+[repository](https://github.com/tensorflow/models/tree/master/official/r1/resnet).
+
+### Steps to Reproduce Conversion to Frozen Graph
+
+1. Install TensorFlow\*, version lower than 2.0.0
```

Comment:
I was able to make the script work with TF 2 (2.1.0, to be exact) simply by changing the TensorFlow import to `import tensorflow.compat.v1 as tf`. I think you should do the same and then remove the version requirement.

Reply:
After an IRL discussion, we decided that for reproducibility, it would be better to specify the exact version of TensorFlow that the model was converted with.